src/Rings/AlgClosureFp.jl

```diff
@@ -101,11 +115,11 @@ end
 //(a::AlgClosureElem, b::AlgClosureElem) = AlgClosureElem(op(//, a, b), parent(a))
 divexact(a::AlgClosureElem, b::AlgClosureElem) = AlgClosureElem(op(divexact, a, b), parent(a))
 ==(a::AlgClosureElem, b::AlgClosureElem) = op(==, a, b)
-Oscar.iszero(a::AlgClosureElem) = iszero(data(a))
-Oscar.isone(a::AlgClosureElem) = isone(data(a))
+iszero(a::AlgClosureElem) = iszero(data(a))
```

Comment:
Does this work? as it needs to extend THE iszero...

Reply:
No, but they have to be imported. I had sometimes surprising results in submodules. No preference either. If it works, I am good